### ISSUE TITLE
MetaVelvet seems stuck during Scaffolding step

### ISSUE THREAD

**Issue by GiBacci** [hacchy/MetaVelvet#2]
Hi,
I'm trying to use MetaVelvet for assembling a small metagenome. I have followed the pipeline reported at [http://metavelvet.dna.bio.keio.ac.jp/MV.html](http://metavelvet.dna.bio.keio.ac.jp/MV.html) but when I have run MetaVelvet the first time it has stopped at the `[VelvetGraph]  === Scaffolding ===` step without writing any new file. Currently it is stuck for almost 24 hours and I'm wondering what to do...
Have you any ideas?

Thanks in advance

Giovanni

**Comment by vmon5589** [comment]
Yes, mine freezes also at the scaffolding step, specifically "Starting pebble resolution"
I am on a mac, I thought it was my compiler, but I have tried updating g++/gcc, re-compiling, and have had no success.

[meta-velvetg] Scaffolding based on paired-end information ...
[MetaGraph]  === Scaffolding with single peak mode ===
[VelvetGraph]  === Rock Bank  ===
[12.903121] Read coherency...
[12.903600] Identifying unique nodes
[12.904020] Done, 764 unique nodes counted
[12.904026] Trimming read tips
[12.904530] Renumbering nodes
[12.904535] Initial node count 39563
[12.904569] Removed 0 null nodes
[12.904572] Confronted to 0 multiple hits and 0 null over 0
[12.904574] Read coherency over!
[VelvetGraph]  === Create read paring array ===
[VelvetGraph]  === Detach dubious reads ===
[VelvetGraph]  === Activate gap markers ===
[VelvetGraph]  === Scaffolding ===
[12.905914] Starting pebble resolution...

**Comment by alfbenpa** [comment]
Hi, I got the same errors and there are a lot of users with similar issues on different blogs on the web with no response from anybody. Metavelvet authors seem to not want replying at all!

[meta-velvetg] Scaffolding based on paired-end information ...
[MetaGraph]  === Scaffolding with single peak mode ===
[VelvetGraph]  === Rock Bank  ===
[676.108468] Read coherency...
[676.135776] Identifying unique nodes
[676.158837] Done, 282868 unique nodes counted
[676.158863] Trimming read tips
[676.204137] Renumbering nodes
[676.204159] Initial node count 628986
[676.206006] Removed 0 null nodes
[676.206010] Confronted to 0 multiple hits and 0 null over 0
[676.206012] Read coherency over!
[VelvetGraph]  === Create read paring array ===
[VelvetGraph]  === Detach dubious reads ===
[VelvetGraph]  === Activate gap markers ===
[VelvetGraph]  === Scaffolding ===
[677.420767] Starting pebble resolution...
Segmentation fault (core dumped)

Alfonso

**Comment by yanhGitHub** [comment]
I am facing the same problem, "Starting pebble resolution...
Segmentation fault (core dumped)". But the memory was only used 1%, why the core dumped with metavelvet?
I really forward to figure this problem.
